sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/ExceptionExtensions.cs

```diff
@@ -24,7 +24,7 @@ internal static string ToInvariantString(this Exception exception)
             }
         }
 
-        internal static Exception? LogAsyncException(this Exception exception)
+        internal static Exception LogAsyncException(this Exception exception)
```

Comment:
Rename this (this method does not "Log" anything).

Reply:
I agree.
I'm going to spin this off into a separate PR.
This method also doesn't have adequate test coverage, and I don't want THIS PR to get blocked on this.